src/bridge-validator-types.ts

```diff
@@ -0,0 +1,169 @@
+export type AnySchema =
```

Comment:
We should not be manually pulling the types like this. We can discuss in our call this afternoon how to better extract these types . Otherwise if the package changes it's types in a version update, we might have some drift between this file and the real types.